### PR TITLE
Disable portaudio on Windows

### PR DESCRIPTION
**Pull request by Umcaruje**
#512 Has been happening for more and more users lately, we have people who can't even open the software on their machines. I provided them with a build with portaudio removed and their issues were gone.

Portaudio was always hard to set up on windows for me, and I find SDL works every time. If WASAPI support is required, I think we should focus on libsoundio, even though that would mean dropping support for windows XP. ASIO is also a planned feature for libsoundio: https://github.com/andrewrk/libsoundio/issues/23

Anyways, in conclusion, Portaudio causes more problems than benfits on windows, and I think it should be disabled.

### CODE REVIEW

**Comment by Umcaruje** [comment]
Unless there are objections, I'll merge this in 24 hours.

**Comment by lukas-w** [comment]
It's easy enough to revert in case there are any objections. 👍

File: CMakeLists.txt [hunk 1]
```diff
@@ -85,12 +85,14 @@ IF(LMMS_BUILD_WIN32)
...
 	SET(WANT_JACK OFF)
 	SET(WANT_PULSEAUDIO OFF)
+	SET(WANT_PORTAUDIO OFF)
 	SET(WANT_SOUNDIO OFF)
 	SET(WANT_WINMM ON)
 	SET(LMMS_HAVE_WINMM TRUE)
 	SET(STATUS_ALSA "<not supported on this platform>")
 	SET(STATUS_JACK "<not supported on this platform>")
 	SET(STATUS_PULSEAUDIO "<not supported on this platform>")
+	SET(STATUS_PORTAUDIO "<disabled on this platform")
```
Comment (PhysSong) [inline CMakeLists.txt]: Missing `>` here.

Comment (lukas-w) [reply CMakeLists.txt]: [Fixed](https://github.com/LMMS/lmms/commit/20ea945cee6030cb5872ff27c095976c64eaf631)

**Pull request merged**